pimd/pim_tlv.c

```diff
@@ -252,8 +261,16 @@ uint8_t *pim_tlv_append_addrlist_ucast(uint8_t *buf, const uint8_t *buf_pastend,
 		struct prefix *p = ifc->address;
 		int l_encode;
 
+#if PIM_IPV == 4
```

Comment:
Given this output:
sharpd@eva ~/f/zebra (master)> vtysh -c "show int dummy9"
Interface dummy9 is up, line protocol is up
  Link ups:       0    last: (never)
  Link downs:     0    last: (never)
  vrf: default
  index 80 metric 0 mtu 1500 speed 0 
  flags: <UP,BROADCAST,RUNNING,NOARP>
  Type: Ethernet
  HWaddr: 5a:b1:91:9c:3a:89
  inet 192.168.4.1/24
  inet 192.168.5.1/24
  inet 192.168.6.2/24
  inet 192.168.6.3/24 secondary
  inet 192.168.6.1/24 secondary
  inet6 fe80::58b1:91ff:fe9c:3a89/64
  inet6 1::2/64
  inet6 1::3/64
  inet6 2::3/64
  inet6 3::3/64
  Interface Type Other
  Interface Slave Type None
  protodown: off 


Why do we need separate code for v4 / v6 paths?

Reply:
Addressed.